articles/appliance/index.md

```diff
@@ -2,57 +2,57 @@
+The Auth0 appliance can be deployed in either Auth0's cloud, your cloud, or your own datacenter. If deploying on-premises in your own datacenter, you are responsible for the operation and monitoring of the servers that host the Auth0 service.
 
-[This article](/deployment) details the main differences between the 4 deployment models.
+See [Auth0 Deployment Models](/deployment) for details on the main differences between each of the four deployment models.
 
-###Who installs Auth0?
+###Auth0 installation and setup
 
-Initial deployment, setup, configuration, security patches, software upgrades, and overall maintenance are responsibilities of Auth0. These activities happen on your infrastructure (e.g. your network, servers, etc.), therefore Auth0 closely coordinates access to it with your operations teams.
+The initial deployment, setup, configuration, security patches, software upgrades, and overall maintenance of the Auth0 appliance are the responsibilities of Auth0. Since all these activities run on your infrastructure (e.g. your network, servers, etc.), Auth0 must closely coordinate access with your operations team.
 
-###How many servers are needed?
+###Server requirements
 
-This depends on the expected traffic and availability required. In some scenarios, a single node is sufficient. The minimum highly available deployment requires three network load balanced nodes. No special load balancing logic is needed (e.g. no sticky sessions).
+The number of servers required to host the Auth0 appliance depends on the expected traffic and the desired availability. In some scenarios, a single node is sufficient. The minimum requirement for a highly available deployment is three load balanced network nodes. No special load balancing logic is needed (no sticky sessions).
 
-###What kind of network access does Auth0 need?
+###Internet access
 
-Depending on the features used in the platform, you might need to permit access to certain websites on the Internet from each of nodes. For example, if Facebook logins are needed, you will have to open connectivity from the Auth0 servers to `facebook.com`. If users are expected to login from the Internet with no VPN, then the authentication endpoints must be published.
+Depending on which features are implemented, you may need to permit access to certain websites on the internet from each node. For example, if Facebook logins are needed, you must open connectivity from the Auth0 servers to `facebook.com`. If users will be logging in from the internet (and not through a VPN), authentication endpoints must be published.
 
-The diagram below details some of these dependencies:
+The diagram below details a few of these dependencies:
 
 ![](/media/articles/appliance/overview.png)
 
-###Will Auth0 nodes deployed on-premises "call home"?
+###On-premises deployment communication with Auth0 
```

Comment:
I would change this to "Communication with Auth0", since it also applies to deployments in your cloud (not only on-premises)